StockSyncAdapter/app/src/main/java/shuvalov/nikita/stocksyncadapter/StockRecyclerAdapter.java

```diff
@@ -0,0 +1,103 @@
+package shuvalov.nikita.stocksyncadapter;
+
+
+import android.database.Cursor;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import java.util.List;
+
+/**
+ * Created by NikitaShuvalov on 12/8/16.
+ */
+
+public class StockRecyclerAdapter extends RecyclerView.Adapter{
+    List<Stock> mStockPortfolio;
+
+    public StockRecyclerAdapter(List<Stock> stockPortfolio) {
+        mStockPortfolio = stockPortfolio;
+    }
+
+    @Override
+    public RecyclerView.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        if(viewType==0){
+            View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.viewholder_stock,null);
+            return new StockViewHolder(view);
+        }else{
+            View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.viewholder_nasdaq, null);
+            return new NasdaqViewHolder(view);
+        }
+    }
+
+    @Override
+    public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {
+        if(holder.getItemViewType()==1){
```

Comment:
I recommend using constants rather than raw ints for view types, but otherwise good use of multiple ViewHolders!